### DISCUSSION TITLE
Modification Method of Reaction in ADC Model Construction

### DISCUSSION THREAD

**Discussion by csungeun** [Open-Systems-Pharmacology/Forum#1584] [Questions & Problems]
hi,

I'm practicing with a tutorial that builds ADC models. (ADC_test.pksim5)

From what I see, I think we proceeded with the pdf(ISoP webinar_ADC modeling_Step-by-Step.pdf)&mbp3(Ex_1_ADC_Tox_Modeling_8.mbp3) without any typos, but the part circled in yellow is different.
![ADC](https://github.com/Open-Systems-Pharmacology/Forum/assets/75017721/0053411e-e325-476f-bdfe-4f3d2d2f2dc7)

Or is there a video of this pdf for my reference?

[Uploading ADC.zip…]()

**Comment by wilbertdew** [comment]
Hi @csungeun, at https://training-esqlabs.talentlms.com for industry or https://learn-esqlabs.talentlms.com for academia (free), you can find a more recent version of this training. I dont know this version of the training that you mention, but the part you indicated is indeed problematic, you have created the drug-target complex twice. Instead, you should connect the internalization reaction to the molecule that is formed by the binding reaction, i.e. ADC-TMEFF2 Boswell et al. Complex, in your example.
